pom.xml

```diff
@@ -86,12 +86,13 @@
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-text</artifactId>
-        <version>1.9</version>
+        <version>1.10.0</version>
       </dependency>
       <dependency>
         <groupId>org.eclipse.jgit</groupId>
         <artifactId>org.eclipse.jgit</artifactId>
-        <version>5.13.0.202109080827-r</version>
+        <!-- Starting with 6.*, Java 11 is the minimum version which we can't use yet. -->
```

Comment:
(optional)
I found the comment hard to understand at first due to missing punctuation.
Something like one of these might be clearer?

"Starting with 6.*, Java 11 is the minimum version, which we can't use yet."

or

"Starting with 6.*, Java 11 is the minimum version (which we can't use yet)."

or

"We have to use version 5.* since it's the last to support Java 8."

or

"TODO: Update to version 6+ once we drop support for Java 8"